app/src/main/java/com/github/sdpsharelook/storage/RTDBWordListRepository.kt

```diff
@@ -15,9 +16,9 @@ import javax.inject.Inject
 class RTDBWordListRepository @Inject constructor(
     private val firebaseDatabase: FirebaseDatabase,
     private val auth: AuthProvider
-) : IRepository<List<@JvmSuppressWildcards Word>> {
-
-    private val reference: DatabaseReference by lazy { firebaseDatabase.reference.child("wordlists") }
+) : IRepository<@JvmSuppressWildcards Word> {
```

Comment:
Noooooooooooooooo my tests

Also@JvmSuppressWildcards is no longer necessary here